package.json

```diff
@@ -143,13 +143,13 @@
     "eslint-plugin-react-hooks": "^1.7.0",
     "file-loader": "^1.1.11",
     "html-webpack-plugin": "^4.5.0",
-    "husky": "^3.0.5",
+    "husky": "^4.3.0",
     "jest": "^24.8.0",
     "jest-styled-components": "^6.3.1",
-    "jest-useragent-mock": "^0.0.3",
+    "jest-useragent-mock": "^0.1.1",
     "jsdom": "^13.0.0",
     "lint-staged": "^8.2.1",
-    "mkdirp": "^0.5.1",
+    "mkdirp": "^1.0.4",
```

Comment:
I had problems with mkdirp as the interface changed and the documentation is a bit of a mess - I ended up removing this and using what we have in use-of-force: https://github.com/ministryofjustice/hmpps-book-video-link/pull/46

Reply:
(The problem was that it broke the build info scripts)

Reply:
Awesome, thanks Andy.